crates/hdk/ON-WASM.md

```diff
@@ -0,0 +1,551 @@
+
```

Comment:
You can ignore this files "changes"... I didn't change anything just copied it verbatim from the original.